dz-9/src/main/java/org/example/Man.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+public class Man extends Person {
+    private String partnerLastName;
```

Comment:
This field redundant here

Reply:
The field partherLastName deleted (row 4)